pkg/helm/helm.go

```diff
@@ -299,6 +308,13 @@ func (r *Impl) GetConfig(kubeNamespace string, log action.DebugLog) (*action.Con
 	config := &genericclioptions.ConfigFlags{
 		Namespace: &kubeNamespace,
 	}
+	if r.discoveryBurst != -1 {
```

Comment:
I think it's better to refer to positive values only: **> 0**

Reply:
the `GetDiscoveryBurst` and `GetDiscoveryQPS` are defined that they return `-1` if something wrong, so I think checking `-1` is appropriate here.